src/cds_resource.h

```diff
@@ -166,6 +168,7 @@ class CdsResource {
         { CdsResource::Attribute::LANGUAGE, "dc:language" },
         { CdsResource::Attribute::AUDIOCODEC, "sec:acodec" },
         { CdsResource::Attribute::VIDEOCODEC, "sec:vcodec" },
+        { CdsResource::Attribute::FILEFORMAT, "fileFormat" },
```

Comment:
fileFormat here would be DSD if I am reading it right, but the file on disk is actually a wavpack?

IMO We need to _store_ 3 bits of info:
`ContainerType`, `VideoCodec` and `AudioCodec`

Here I guess ContainerType is WavPack, and AudioCodec is DSD?

Reply:
It's more complicated here.

Wavpack is some sort of ZIP file that contains compressed audio (no video). The audio file can be WAV, DFF or DSD, etc. The audio like DSD may use a codec. So we cannot store DSD it as acodec - vcodec was just the hack to get dlnaprofile selection tested. `fileFormat` describes the media type of packed file.

Reply:
I think we are agreeing 🤔 

We need to know the format inside, which is what the acodec field is for in my understanding.

In my head it looks like
Typical MKV video `mkv(h264,aac)`: acodec=aac, vcodec=h264 and fileformat=mkv
OGV file `ogg(h264,mp3)`: acodec=mp3, vcodec=h264 and fileformat=ogg
OGA file `ogg(null,vorbis)`: acodec=vorbis, vcodec=null and fileformat=ogg
M4A file `mp4(null,aac)`: acodec=aac, vcodec=null and fileformat=mp4

So with wavpack: `wv(null,dsd)`: acodec=dsd, vcodec=null and fileformat=wv

We need all 3 bits of information at least for transcoding effectively.

Say we have a client that supports DSD audio streams but only in its "native" DSF container, we could create a profile that simply copies that stream out of the wavpack with ffmpeg into the new container for that client.

Reply:
Almost audio codec is e.g. lsbf_planar (the codec of the original file) and fileformat is DSD. fileFormat can be content type for other media. So it may happen that we need to transcode because the client does not support lsbf_planar inside a DSD inside a wavpack...

Reply:
As we are already below Resource we can just use `format`